ra/SAPHanaTopology

```diff
@@ -575,21 +572,17 @@ function sht_init() {
         #super_ocf_log debug "DBG2: hdbANSWER=$hdbANSWER"
         #srmode=$(echo "$hdbANSWER" | awk -F= '/mode/ {print $2}')
         case "$chkMethod" in
-            gP ) # call getParameter (gP)
-                local gpKeys=""
+            gP ) # call getParameter (gP) - fallback for 'hdbnsutil' failing 3 times.
+                local gpKeys=''
                 gpKeys=$(echo --key=global.ini/system_replication/{actual_mode,mode,site_name,site_id})
                 hdbANSWER=$(HANA_CALL --timeout "$HANA_CALL_TIMEOUT" --cmd "python getParameter.py $gpKeys --sapcontrol=1" 2>&1 | awk -F/ 'BEGIN {out=0} /^SAPCONTROL-OK: <begin>/ { out=1 } /^SAPCONTROL-OK: <end>/ { out=0 } /=/ {if (out==1) {print $3} }')
-                srmode=$(echo "$hdbANSWER" | awk -F= '$1=="actual_mode" {print $2}')
                 # if 'actual_mode' is not available, fallback to 'mode'
-                if [ -z "$srmode" ]; then
-                    srmode=$(echo "$hdbANSWER" | awk -F= '$1=="mode" {print $2}')
-                fi
+                [[ "$hdbANSWER" =~ (actual_mode|mode)=([^$'\n']+) ]] && srmode=${BASH_REMATCH[2]}
```

Comment:
same concern as in https://github.com/SUSE/SAPHanaSR/pull/159 regarding change of output order from 'getParameter.py'

Reply:
@angelabriel Maybe it would be good to catch both values separately (e.g. $actual_mode and $mode) and then to decide which one has "higher" priority.

Reply:
see see https://github.com/SUSE/SAPHanaSR/pull/196 for the changes.
as discussed in #159